### PR TITLE
removed required status check

### PR DESCRIPTION
**Pull request by Falpangaea**

### CODE REVIEW

File: .idea/encodings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tfhartmann) [inline .idea/encodings.xml]: Lets add all references to .idea to the .gitignore file

File: check.go [hunk 2]
```diff
@@ -135,8 +135,9 @@ func checkUrl(u urlCheck, c *http.Client) (urlCheck, error) {
 	u.Checked = true
 
 	switch {
-	case u.WantStatus != u.GotStatus:
-		u.Valid = false
+	/* If you want to verify the status as well in order to return 'success', uncomment out the following two lines */
```
Comment (tfhartmann) [inline check.go]: I think rather then just comment these out, I suggest we add an argument on the command line to allow us to skip them.

Comment (tfhartmann) [reply check.go]: oh cc'ing @asicsdigital/devops for other opinions too!

**Comment by tfhartmann** [comment]
@hakamadare I'd love to have your opinion as well! :)

**Comment by tfhartmann** [comment]
Oh! Also lets swap this our for `asicsdigital/hermes:stable` https://github.com/asicsdigital/httpeimdall/blob/master/.circleci/config.yml#L5

File: check.go (hunk 2 above)
Comment (Falpangaea) [reply check.go]: Done

**Pull request merged**